.github/pnnl-ci/rebuild-haero.sh

```diff
@@ -98,6 +98,7 @@ echo "Cloning Haero repository into $(pwd)/.haero..."
 # Need to clone HAERO using HTTPS instead of SSH
 git clone https://github.com/eagles-project/haero.git .haero || exit
 cd .haero || exit
+git checkout jaelynlitz/deception-upgrade || exit
```

Comment:
remove this once https://github.com/eagles-project/haero/pull/485 is merged and test the haero rebuild pipeline again